Enable local builds (much faster than using docker for development)

rpocklin opened the pull request:
Updated the docs, happy to amend if there are any suggestions, this shouldn't differ the final output from the `make build` command but it is much faster for local development of tests.

Also solves the annoying issue of the `tests/k6` folder being owned by root when using `make build` (via docker).

fschade commented on README.md:
```diff
@@ -14,8 +14,12 @@ Supported clouds are:
+$ # build via docker
 $ make clean build
 $
+$ # alternatively, build locally (requires go, k6, node.js + yarn installed)
```
only node/yarn is needed to build it, then running it locally would need go && k6 to be installed

rpocklin replied:
Yes, you are correct, I will make this clearer in the docs.

fschade commented on Makefile:
```diff
@@ -6,6 +7,15 @@ build:
 	docker run --rm --volume $(ROOT_DIR)/tests/k6:/cp owncloud/cdperf-k6 cp -r . /cp
 	chmod +x ./scripts/cdperf
 
+.PHONY: local
```
can you change it to
```
.PHONY: local
local: clean
	cd src/k6 && yarn && yarn build
	mkdir -p tests/k6
	cp -R src/k6/dist/. tests/k6/
```

rpocklin replied:
Sure that works, will change.

fschade reviewed (changes requested):
i updated a few things, can you please have a look. Thanks for your PR

rpocklin commented:
> i updated a few things, can you please have a look. Thanks for your PR

Thanks @fschade - I have made all the changes you requested (and rebased to `main`).

fschade reviewed (approved):
LGTM 🚀  - thanks for the pr 😚
